test/show.jl

```diff
@@ -58,10 +58,12 @@ function test_repr(x::String)
     # strings get converted to string Exprs by the first show().
     # This could produce a few false positives, but until string
     # interpolation works we don't really have a choice.
+    #
+    # Rectification: comparing x1 and x2 seans to be working
```

Comment:
Is it possible to remove the comment as a whole then? I must admit I find the original context of this a bit confusing.

Reply:
Yes, I think it could be removed, but for now I left it there because I'm somehow confused too about this.
Maybe @JeffBezanson  could provide some thought on this matter? (and also about nested quotes and interpolations)